src/import_whatsapp.js

```diff
@@ -160,41 +165,33 @@ const processGeoJson = (json) => {
 	return [mapdata, groupName];
 };
 
-const processText = (text) => {
-	const groupNameRegex = /"([^"]*)"/;
-	const groupNameMatches = text.match(groupNameRegex);
-	const groupName = groupNameMatches ? groupNameMatches[1] : null;
-
-	// Check the first 3 characters to determine the format; iOS and Android
-	const fileType = text.substring(0, 3);
-	let messageRegex;
-
-	// Regex matches a single message including newline characters,
-	// stopping when new line starts with date or text ends
-	// also accounts for if the datetime is wrapped in brackets and has s
-	// Capture group 1 = date, group 2 = time, group 3 = sender, group 4 = message content
-	// this has been tweaked for each format but gives the same output
-	if (fileType.match(/\[\d{2}/)) {
-		console.info("ios format");
-		// iOS format
-		messageRegex =
-			/\[(\d{2}\/\d{2}\/\d{4}),\s(\d{1,2}:\d{2}:\d{2}\s(?:AM|PM))\]\s(.*?):\s(.+?)(?=\n\[|$)/gs;
-	} else if (fileType.match(/\d{2}\//)) {
-		console.info("android format");
-		// Android format
-		messageRegex =
-			/(\d{2}\/\d{2}\/\d{4}),?\s(\d{1,2}:\d{2})(?:\s?(?:AM|PM|am|pm))?\s-\s(.*?):[\t\f\cK ]((.|\n)*?)(?=(\n\d{2}\/\d{2}\/\d{4})|$)/g;
-	} else {
-		console.error("Unknown file format");
-		return [null, groupName];
-	}
-
-	let messageMatches = [...text.matchAll(messageRegex)];
-	// Regex to match google maps location and capture lat (group 1) and long (group 2)
-	const locationRegex =
-		/: https:\/\/maps\.google\.com\/\?q=(-?\d+\.\d+),(-?\d+\.\d+)/g; //Without 'location' to be universal - the word in the export file changes based on WA language
+const cleanMsgContent = (content, location, imgFileRegex) => {
```

Comment:
Could you add some comments to explain what the purpose of this function is. In particular I can't tell just by looking what the location and imgFileRegex bits are doing. Is the purpose to just strip out the location and filename from the content, but keep the rest?